src/pymodaq/extensions/bayesian/bayesian_optimisation.py

```diff
@@ -58,58 +60,49 @@ class BayesianOptimisation(CustomExt):
-                   'tip': 'Parameter to indicate how closed are the next parameters sampled.'
-                          'Higher value = favors spaces that are least explored.'
-                          'Lower value = favors spaces where the regression function is the '
-                          'highest.'},
-                  {'title': 'Kappa actual:', 'name': 'kappa_actual', 'type': 'float', 'value': 2.576,
-                   'tip': 'Current value of the kappa parameter', 'readonly': True},
-                  {'title': 'xi:', 'name': 'xi', 'type': 'slide', 'value': 0,
-                   'tip': 'Governs the exploration/exploitation tradeoff.'
-                          'Lower prefers exploitation, higher prefers exploration.'},
-                  {'title': 'Kappa decay:', 'name': 'kappa_decay', 'type': 'float', 'value': 0.9,
-                   'tip': 'kappa is multiplied by this factor every iteration.'},
-                  {'title': 'Kappa decay delay:', 'name': 'kappa_decay_delay', 'type': 'int',
-                   'value': 20, 'tip': 'Number of iterations that must have passed before applying '
-                                      'the decay to kappa.'},
-              ]},
-             {'title': 'Stopping Criteria:', 'name': 'stopping', 'expanded': False, 'type': 'group',
-              'children': [
-                  {'title': 'Niteration', 'name': 'niter', 'type': 'int', 'value': 100, 'min': -1},
-                  {'title': 'Type:', 'name': 'stop_type', 'type': 'list',
-                   'limits': StopType.names()},
-                  {'title': 'Tolerance', 'name': 'tolerance', 'type': 'slide', 'value': 1e-2,
-                   'min': 1e-8, 'max': 1, 'subtype': 'log',},
-                  {'title': 'Npoints', 'name': 'npoints', 'type': 'int', 'value': 10, 'min': 1},
-              ]},
-             {'title': 'Ini. State', 'name': 'ini_random', 'type': 'int', 'value': 10},
-             {'title': 'bounds', 'name': 'bounds', 'type': 'group', 'children': []},
-         ]},
-
-        {'title': 'Models', 'name': 'models', 'type': 'group', 'expanded': True, 'visible': True,
-         'children': [
-            {'title': 'Models class:', 'name': 'model_class', 'type': 'list',
-             'limits': [d['name'] for d in models]},
-            {'title': 'Ini Model', 'name': 'ini_model', 'type': 'action', },
-            {'title': 'Ini Algo', 'name': 'ini_runner', 'type': 'action', 'enabled': False},
-            {'title': 'Model params:', 'name': 'model_params', 'type': 'group', 'children': []},
-        ]},
-        {'title': 'Move settings:', 'name': 'move_settings', 'expanded': True, 'type': 'group',
-         'visible': False, 'children': [
-             {'title': 'Units:', 'name': 'units', 'type': 'str', 'value': ''}]},
-
-    ]
+
+    acquisition_functions_names = list(GenericAcquisitionFunctionFactory.keys())
+
 
     def __init__(self, dockarea, dashboard):
+        self.params = [
```

Comment:
why is it here and not in the class preamble? The main purpose to have it in the class preamble is for introspection and settings generation even if the object is not instanciated!